### DISCUSSION TITLE
Can't display experiment in the middle of the webpage

### DISCUSSION THREAD

**Discussion by viviantic** [jspsych/jsPsych#1034] [Q&A] [answered]
Hello everyone! 

I don't know if this had occurred to you, I have some issues displaying my experiment. My script is in Mandarin Chinese, so the test stimuli and the instructions contain Chinese characters. Everything seemed to work well using whatever browser. The experiment is also correctly displayed (right in the middle of the webpage) when I run the experiment locally, but when I publish it on my school server, everything was shifted to the upper left corner of the webpage as seen below: 
![Screen Shot 2020-09-07 at 3 58 29 PM](https://user-images.githubusercontent.com/65309104/92414753-aefa2580-f123-11ea-8762-249e54ac8a8a.png)

![Screen Shot 2020-09-07 at 3 58 50 PM](https://user-images.githubusercontent.com/65309104/92414757-b15c7f80-f123-11ea-884f-8617c10b8bf5.png)

Any suggestions on how to solve this problem and move everything back to the middle?

**Comment by jodeleeuw** [comment, marked as answer]
Is the jspsych.css file loading properly on the server version? You should see an error in the console if the file is not found.

**Reply by viviantic** [reply]
No, no errors. I can run the experiment smoothly with no problem, except that everything is displayed in the upper left corner.

**Reply by jodeleeuw** [reply]
Have you confirmed that the CSS file is loading? If you look at the elements tab of the debugger you should see that CSS rules from jspsych.css are being applied:

![image](https://user-images.githubusercontent.com/595524/92415549-76f4e180-f127-11ea-8041-11a11dbf0e96.png)

**Reply by viviantic** [reply]
I think the CSS file is loaded and the rules are applied. Here's what I got: 
![Screen Shot 2020-09-07 at 4 41 18 PM](https://user-images.githubusercontent.com/65309104/92415907-fd5df300-f128-11ea-8e00-b83c66f36228.png)

**Reply by jodeleeuw** [reply]
It looks like there aren't any CSS rules (on the right-hand side) from jspsych.css. Check the `<head>` section of your experiment and make sure that the link to the CSS file is there and that you've uploaded the CSS file to the corresponding location on your server.

**Reply by viviantic** [reply]
Thanks, @jodeleeuw , for the troubleshooting. I just checked the head section of the experiment as well as the experiment folder. I do have the link to the CSS file in my script, and the CSS file is in the right location on my server. 
![Screen Shot 2020-09-07 at 5 01 23 PM](https://user-images.githubusercontent.com/65309104/92416582-351a6a00-f12c-11ea-9866-38f77d219de5.png)

![Screen Shot 2020-09-07 at 5 07 12 PM](https://user-images.githubusercontent.com/65309104/92416706-c8539f80-f12c-11ea-9adf-9b81797ac11d.png)

Not sure what went wrong at this point. I'll play around and upload the CSS file again to see if it makes any difference. Thank you so much for helping along.

**Reply by jodeleeuw** [reply]
The `//` may be part of the issue, though I'm not confident about that. Comments in HTML are done like this:

```html
<!-- 
	this is commented out
-->
```

**Reply by viviantic** [reply]
Yep. It was the `//`. Can't thank you enough, @jodeleeuw!
